usr/share/rear/verify/COMMVAULT/default/400_start_galaxy_daemons.sh

```diff
@@ -0,0 +1,18 @@
+#
+# start commvault daemons and check that they run and connect to the backup server
+#
+#
+#
+
+# Commvault dies when it finds /lib/tls with libpthread
+# Relax-and-Recover has a symlink from /lib to /lib/tls and I don't remember why (Schlomo)
+
+# delete problematic symlink
+rm -f /lib/tls
+
+/opt/commvault/Base/Galaxy start
+if /opt/commvault/Base/Galaxy list | grep -q N/A ; then
+	Error "Commvault daemon did not start. Please check with '/opt/commvault/Base/Galaxy list'"
```

Comment:
A technical writing style suggestion:

Better avoid personal speech style in technical texts
like computer messages or program documentation and such.

So avoid as if you personally talk to the user like
"please" or "thank you" or "sorry" and so on.

Better keep computer output neutral and don't "touch" the user
for example in this case it could be like
```
Error "Commvault daemon did not start. Check with '/opt/commvault/Base/Galaxy list'"
```
or perhaps even strictly only what the plain facts are like
```
Error "No Commvault daemon ('/opt/commvault/Base/Galaxy list' shows 'N/A')"
```
and leave all interpretation to the user.
The latter is my preferred style but I understand
when this is considered as too hard
(mathematicians love rigor).

Reply:
:heavy_check_mark: